public/stylesheet/pumpio.css

```diff
@@ -253,6 +260,12 @@ body {
   margin-top: 20px;
 }
 
+/* helpers */
+
+.pull-center {
+  text-align: center;
```

Comment:
Why is this needed?

Reply:
for https://github.com/pump-io/pump.io/pull/1486/files#diff-a81d9634aa223a78b1c4e1e3ddad90ecR25, when the width is less than the maximum it looks to right and it looks very weird, that helper fix and looks like the image.